tools/on_device_tests_gateway.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 The Cobalt Authors. All Rights Reserved.
```

Comment:
Do we have CL to bring tools/on_device_tests* files to cobalt_src on GoB?

Reply:
Not sure, I haven't done that

Reply:
Let's do that before submitting this PR.  I am not sure if 20.lts.1+ branch in GoB has precommit, but if it does we might want to have that CL go through precommit.  

FYI @andrewsavage1

Reply:
I created a ticket for tools/on_device_tests on GoB. I will be working on it once all the legacy branches changes are ready to be submitted.